chore: combine mastering and accelerated networking tutorials (#72)
canonical/charmed-aether-sd-core: docs/tutorials/mastering.md

Create [SR-IOV Network Device Plugin] ConfigMap by replacing the `pciAddresses` with the PCI addresses of `access` and `core` interfaces:

```console
cat <<EOF | kubectl apply -f -
apiVersion: v1
kind: ConfigMap
metadata:
  name: sriovdp-config
  namespace: kube-system
data:
  config.json: |
    {
      "resourceList": [
        {
          "resourceName": "intel_sriov_vfio_access",
          "selectors": {
            "pciAddresses": ["0000:07:00.0"]
          }
        },
        {
          "resourceName": "intel_sriov_vfio_core",
          "selectors": {
            "pciAddresses": ["0000:06:00.0"]
          }
        }
      ]
    }

EOF
```

Deploy [SR-IOV Network Device Plugin]:

```console
kubectl apply -f https://raw.githubusercontent.com/k8snetworkplumbingwg/sriov-network-device-plugin/v3.6.2/deployments/sriovdp-daemonset.yaml
```

##### Checkpoint 4: Check the allocatable resources in the Kubernetes node

Make sure that there are 2 `1Gi HugePages`, 1 `intel_sriov_vfio_access` and 1 `intel_sriov_vfio_core` are available by running the following command:

```console
sudo snap install jq
kubectl get node -o json | jq '.items[].status.allocatable'
```

> You should see the following output:

Reviewer: Maybe we can include a note that this might not happen immediately to avoid confusing readers if it takes a bit of time?

Author: Done

terraform apply -auto-approve
```

## 5. Deploy User Plane Function (UPF) in DPDK mode

Create a Juju model named `user-plane`:

```console
juju add-model user-plane user-plane-cluster
```

Deploy `sdcore-user-plane-k8s` Terraform Module.
In the directory named `terraform`, update the `main.tf` file.

> Please replace the `access-interface-mac-address` and `core-interface-mac-address` according your environment. They are advised to be noted in the `Checkpoint 4`.

Reviewer: ```suggestion
Please replace the `access-interface-mac-address` and `core-interface-mac-address` according your environment. You would have noted them at the `Checkpoint 4`.
```

I think this reads a bit better, WDYT?
